### PR TITLE
Python SDK: Allow quoted string config vars

### PR DESCRIPTION
**Pull request by josephaxisa**
This addresses #131

### CODE REVIEW

File: python/looker_sdk/rtl/api_settings.py [hunk 1]
```diff
@@ -114,10 +116,19 @@ def read_ini(filename: str, section: Optional[str] = None) -> Dict[str, str]:
...
+    def _clean_input(cls, config_data: Dict[str, str]) -> Dict[str, str]:
+        for setting, value in list(config_data.items()):
+            # Remove empty setting values
+            if not isinstance(value, str):
+                continue
+            if value in ['""', "''", ""]:
+                config_data.pop(setting)
+            # Strip quotes from setting values
+            elif value.startswith(('"', "'")) or value.endswith(('"', "'")):
+                config_data[setting] = value.strip("\"'")
```
Comment (jkaster) [inline python/looker_sdk/rtl/api_settings.py]: You only want to strip beginning and ending quote values. Does `value.strip()` remove all occurrences?

File: python/tests/integration/test_methods.py [hunk 2]
```diff
@@ -296,7 +296,7 @@ def test_search_look_and_run(sdk: mtds.LookerSDK):
     """run_look() should return CSV and JSON
     CSV will use column descriptions
     JSON will use column names
-    JSON_LABEL will use column descriptions
+    JSON_DETAIL will use column descriptions
```
Comment (jkaster) [inline python/tests/integration/test_methods.py]: Did the behavior of JSON_LABEL change? Wasn't this test intended to verify the behavior of the new format, JSON_LABEL?

File: python/looker_sdk/rtl/api_settings.py (hunk 1 above)
Comment (josephaxisa) [reply python/looker_sdk/rtl/api_settings.py]: It only removes leading/trailing occurrences: https://docs.python.org/3/library/stdtypes.html#str.strip

Comment (jkaster) [reply python/looker_sdk/rtl/api_settings.py]: Ah. Most languages I know call that `trim` of some sort. Thanks.

File: python/tests/integration/test_methods.py (hunk 2 above)
Comment (josephaxisa) [reply python/tests/integration/test_methods.py]: You're right. I just reverted this change.

**Pull request merged**